src/main/webapp/app/entities/alteration/alteration.store.ts

```diff
@@ -18,6 +21,15 @@ export class AlterationStore extends PaginationCrudStore<IAlteration> {
     this.totalItems = result.headers['x-total-count'];
     return this.entities;
   }
+  *annotateAlteration({ geneIds, alteration }) {
+    const alt = {
+      genes: geneIds.map(id => ({ id })),
+      alteration,
+    };
+    const result = yield new AlterationControllerApi(null, '', axiosInstance).annotateAlteration(alt as Alteration);
```

Comment:
I started to put some of the api clients in `shared/api/clients.ts`, so they can be reused. If that's a good idea, then we can move the `AlterationControllerAPI` client into that file as well.